pkg/server.go

```diff
@@ -149,7 +150,10 @@ func index(template *template.Template, counterHandle counterHandler) http.Handl
 func healthz() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if atomic.LoadInt32(&healthy) == 1 {
-			w.WriteHeader(http.StatusNoContent)
+			_, err := io.WriteString(w, "OK\n")
```

Comment:
Please do this without importing `io`:
```suggestion
			err := w.Write([]byte("OK"))
```

Reply:
That was my initial implementation before I had to figure out how to implement error handling in Go. ;)